lambdas/pkgpush/requirements.txt

```diff
@@ -1,2 +1,25 @@
+attrs==23.1.0
+aws-requests-auth==0.4.3
+boto3==1.26.133
+botocore==1.29.133
+certifi==2023.5.7
+charset-normalizer==3.1.0
+idna==3.4
+importlib-metadata==6.6.0
+jmespath==1.0.1
+jsonlines==1.2.0
 jsonschema==3.2.0
-https://github.com/quiltdata/quilt/archive/e96df90afc80ea2e3820a57a2a7ef4477df24204.tar.gz#egg=quilt3&subdirectory=api/python
+platformdirs==3.5.1
+pyrsistent==0.19.3
+python-dateutil==2.8.2
+PyYAML==6.0
+quilt3 @ https://github.com/quiltdata/quilt/archive/e96df90afc80ea2e3820a57a2a7ef4477df24204.tar.gz#subdirectory=api/python
```

Comment:
@sir-sigurd is this still the right pin?

Reply:
This was generated by `pip freeze`, and seems to work the same. But I could revert to the original. Not sure what the "correct" way is.

Reply:
I’m not worried about the syntax just do we really need that weird version or was that a temporary hack and we since updated PyPI?

Reply:
Ohh. Probably don't need that version - I guess if we can specify a normal release number, that would be better. Let me see.

Reply:
https://github.com/quiltdata/quilt/pull/3452